src/Aspire.Hosting.AWS/Utils/ProjectUtilities.cs

```diff
@@ -0,0 +1,184 @@
+        {
+            userProfileEnvironmentVariable = "$HOME";
+        }
+
+        if (path.StartsWith(userProfilePath))
+        {
+            return path.Replace(userProfilePath, userProfileEnvironmentVariable);
+        }
+
+        return path;
+    }
+    
+    /// <summary>
+    /// Retrieve a project's launchSettings.json file contents
+    /// </summary>
+    /// <param name="projectPath">The project file path</param>
+    /// <returns>The launchSetting.json content</returns>
+    private static string GetLaunchSettings(string projectPath)
+    {
+        var parentDirectory = Path.GetDirectoryName(projectPath);
+        if (string.IsNullOrEmpty(parentDirectory))
+            throw new ArgumentException($"The project path '{projectPath}' is invalid. Unable to retrieve the '{Constants.LaunchSettingsFile}' file.");
+        var properties = Path.Combine(parentDirectory, "Properties");
+        if (!Directory.Exists(properties))
+        {
+            return "{}";
+        }
+
+        var fullPath = Path.Combine(properties, Constants.LaunchSettingsFile);
+        if (!File.Exists(fullPath))
+            return "{}";
+
+        return File.ReadAllText(fullPath);
+    }
+
+    /// <summary>
+    /// Write the launchSettings.json content to disk
+    /// </summary>
+    /// <param name="projectPath">The project file path</param>
+    /// <param name="content">The launchSettings.json content</param>
+    private static void SaveLaunchSettings(string projectPath, string content)
+    {
+        var parentDirectory = Path.GetDirectoryName(projectPath);
+        if (string.IsNullOrEmpty(parentDirectory))
+            throw new ArgumentException($"The project path '{projectPath}' is invalid. Unable to retrieve the '{Constants.LaunchSettingsFile}' file.");
+        var properties = Path.Combine(parentDirectory, "Properties");
+        if (!Directory.Exists(properties))
+        {
+            Directory.CreateDirectory(properties);
+        }
+        var fullPath = Path.Combine(properties, Constants.LaunchSettingsFile);
+        File.WriteAllText(fullPath, content);
+    }
+
+    /// <summary>
+    /// Create an executable wrapper project that invokes the specified Lambda function.
+    /// </summary>
+    /// <param name="classLibraryProjectPath">The project path of the class library Lambda function</param>
+    /// <param name="lambdaHandler">The Lambda function handler</param>
+    /// <returns>A project file path of the executable wrapper project</returns>
+    public static string CreateExecutableWrapperProject(string classLibraryProjectPath, string lambdaHandler)
+    {
+        string tempPath = Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString());
+        Directory.CreateDirectory(tempPath);
+
+        var projectContent = $@"
+<Project Sdk=""Microsoft.NET.Sdk"">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>{Constants.ExecutableWrapperProjectTargetFramework}</TargetFramework>
```

Comment:
This shouldn't be a constant. We should use the target framework the Lambda function is using.

Reply:
Done